tutorials/tutorial-8/answers/aie.mlir

```diff
@@ -22,29 +25,30 @@ module @tutorial_8 {
     %tile14 = AIE.tile(1, 4) 
     %tile24 = AIE.tile(2, 4)
 
-    // Declare local memory of tile(2,4) which is shared with tile(1,4)
-    %buf = AIE.buffer(%tile14) { sym_name = "a14" } : memref<256xi32>
+    // Declare local memory of tile(2,4) which is shared with tile(1,4), do not change symbolic name to allow reusing original test.cpp
+    %buf = AIE.buffer(%tile14) { sym_name = "a23" } : memref<256xi32>
 
     // declare 2 kernel functions name "extern_kernel1" and "extern_kernel2"
     // with one positional function argument, in this case mapped to a memref
     func.func private @extern_kernel1() -> ()
     func.func private @extern_kernel2(%b: memref<256xi32>) -> ()
 
-    // Declare shared lock (belonging to tile(2,4), lock ID=1)
-    // %lock23_1 = AIE.lock(%tile23, 1)
+    // Declare shared lock (belonging to tile(2,4), lock ID=1), do not change symbolic name to allow reuse of test.cpp
+
+    %lock14_1 = AIE.lock(%tile14, 1) { sym_name = "lock_13_2" }
 
     // Define core algorithm for tile(1,4)
     // buf[3] = 13
     %core14 = AIE.core(%tile14) {
         // Locks init value is Release 0, so this will always succeed first
-        // AIE.useLock(%lock23_1, "Acquire", 0)
+        AIE.useLock(%lock14_1, "Acquire", 0)
 
 		// %val = arith.constant 13 : i384 
```

Comment:
Should the commented out remnants here be removed?

Reply:
@jackl-xilinx do you want to keep the comments because you would like to remove the external kernels in the future?

Reply:
@denolf Yeah, let's keep the comments for now since this tutorial is currently not in its ideal state but moving the file to answers/aie.mlir and cleaning up the lock was needed.